packaging_automation/README.md

```diff
@@ -0,0 +1,79 @@
+# **Prepare Release Usage**
+
+prepare-release.py script performs the  pre-packaging configurations in citus/citus-enterprise projects.
+
+## Installation
+
+Before using script, you need to make sure that Python > 3.8 is installed in your system.
+
+### Clone Tools Repository
+
+git clone https://github.com/citusdata/tools.git
+
+Enter 'tools' directory
+
+``` console
+cd tools
+```
+
+### Install Required Python Libraries
+
+Verify pip installation 
+
+``` console
+python -m pip --version
+```
+Output should be like following
+
+``` console
+pip 21.1.2 from /home/vagrant/.local/lib/python3.8/site-packages/pip (python 3.8)
+```
+
+If you get error, you should first install pip
+``` console
+sudo apt install python3-pip
+```
+Install the required libraries to execute the script
+``` console
+python -m pip install -r packaging_automation/requirements.txt
+```
+If all the steps above completed successfully , you are ready for script execution
+
+## Script Usage
+
+Script can be used for either major release (i.e. third digit of release is '0' e.g. 10.1.0)  or 
+patch release (i.e. third digit of release is other than '0' e.g. 10.0.4).
+
+### Available flags
+
+**--gh_token:** Personal access token that is authorized to commit citus/citus-enterprise projects. (Required)
+
+**--prj_name:** Project to be released. Allowed values 'citus' and 'citus-enterprise (Required)
+
+**--prj_ver:** Upcoming version to be used for release. should include three level of digits separated by dots, e.g: 10.0.1
+(Required)
+
+**--main-branch:** Branch to be used as base to be used for configuration changes. There is no need for base scenario. 
```

Comment:
should be `--main_branch` according to code, not `--main-branch`

Reply:
Fixed